vlingo-e2e-sys-airline-ops/frontend/src/stores/index.js

```diff
@@ -33,7 +33,7 @@ export function createLocalStore(key, initialValue) {
 	};
 }
 
-export const theme = createLocalStore('theme', 'light');
```

Comment:
`createLocalStore` made sense, right? :)
probably the others will get some localstorage, too, so you can revert this one.

Reply:
name is updated